src/XNA/SpriteBatchRenderer.cs

```diff
@@ -63,6 +63,35 @@ public void Draw(ITexture2D texture, Rectangle dest, Rectangle source, Color col
 				SpriteEffects.None,
 				ImageOrientation.AsIs,
 				depth);
+#endif
+		}
+
+		public void Draw(ITexture2D texture, Vector2 position, Rectangle? sourceRectangle, Color color, float rotation,
+			Vector2 origin, Vector2 scale, SpriteEffects effects, float depth)
+		{
+			var textureWrapper = (Texture2DWrapper)texture;
+
+#if MONOGAME || FNA
+			_batch.Draw(textureWrapper.Texture,
+				position,
+				sourceRectangle,
+				color,
+				rotation,
+				origin,
+				scale,
+				SpriteEffects.None,
+				depth);
+#elif STRIDE
+			_batch.Draw(textureWrapper.Texture,

```

Comment:
I don't have/use Stride, so I"m not positive if this is right.  Actually I see an error I'll need to fix.